src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,19 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. The field `dippingDepth` should be accessed through special methods (getters and setters), so it should be private.